CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### 2.1.1 (2023-10-01)
+
+* Restores publishing of source distribution (#139)
+* Fix url in Issue tracker (#140)
+* Version 2.1.1 (#143)
+* Update CHANGELOG.md for 2.1.0 (#142)
```

Comment:
```suggestion
```

Reply:
No - this was already present. Agreeing to it...